backend/src/main/java/com/roc/app/user/UserService.java

```diff
@@ -0,0 +1,26 @@
+package com.roc.app.user;
+
+import com.roc.app.user.dto.UserResponseDto;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class UserService {
+    private final UserRepository userRepository;
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public List<UserResponseDto> getAllUsers() {
+        return userRepository.findAll().stream().map(user ->
+                new UserResponseDto(
+                        user.getUserId(),
+                        user.getEmail(),
+                        user.getFirstName(),
+                        user.getLastName()
```

Comment:
Small detail, as `UserResponseDto` takes email as last argument:
```suggestion
                        user.getFirstName(),
                        user.getLastName(),
                        user.getEmail()
```